.gitignore

```diff
@@ -11,7 +11,7 @@ Cargo.lock
 # Python files
 *.egg-info
 __pycache__
-/env
+env
```

Comment:
Why was this added before?

Reply:
https://github.com/github/gitignore/blob/752b7678db497edb377fa982ded0c70035393794/Python.gitignore#L114-L121

Reply:
We should copy all of them since not everyone uses the `env` name. I prefer to use `venv`.